docs/pprabhala.md

```diff
@@ -0,0 +1,4 @@
+# Bio for @pprabhala
+
+- **Name:** : P X
+- **Location:** : TT City
```

Comment:
I would :heart: to see your occupation here.  You can even have something fake here since it's a public repository. :smiley: